### DISCUSSION TITLE
要怎麼挖出比特幣？

### DISCUSSION THREAD

**Discussion by dabbie109303501** [1101-GS4719-C/Python-Community-GS4719-C#710] [科技新知]
之前在準備科技英文簡報的ppt時發現了這個影片跟優秀的YouTube頻道
推薦給大家
https://youtu.be/bBC-nXj3Ng4

**Comment by Yunhans** [comment]
我只知道要用顯卡

**Comment by Chiway1229** [comment]
看完這個影片後，了解到挖礦需要吃很多顯卡效能，挖礦後的卡差不多已經半死不活了
難怪大家都說不要買礦卡就是這個原因

**Comment by danwan1027** [comment]
現在能挖的礦越來越少，除非找到好的新標地才有可能透過挖礦致富

**Comment by WEI-CHEN-CHUN** [comment]
挖礦非常耗損地球能源

**Comment by williamyen0623** [comment]
現在區塊鏈給予礦工的紅利越來越少（智能合約內容限制，例如：比特幣
用高算力（高電力）去挖礦已成費時、費電、不環保的活動
但區塊鏈的發展仍然令人期待，我相信一定會有人發明更有效率、更新穎的技術來將區塊鏈發揚光大
